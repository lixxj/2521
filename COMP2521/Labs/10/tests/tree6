1000 100 99 33 23 13 10 9 7 6 3 2 1
