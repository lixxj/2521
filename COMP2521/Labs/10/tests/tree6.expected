Tree:
                      1000
                       /
                     100
                     /
                    99
                   /
                  33
                 /
                23
               /
              13
             /
            10
           /
          9
         /
        7
       /
      6
     /
    3
   /
  2
 /
1

#nodes = 13,  #leaves = 1
Infix  : 1 2 3 6 7 9 10 13 23 33 99 100 1000 
Prefix : 1000 100 99 33 23 13 10 9 7 6 3 2 1 
Postfix: 1 2 3 6 7 9 10 13 23 33 99 100 1000 
ByLevel: 1000 100 99 33 23 13 10 9 7 6 3 2 1 
